frameworks/helloworld/tests/test_region_awareness.py

```diff
@@ -60,19 +79,11 @@ def test_nodes_can_deploy_to_remote_region():
             'Europe'
         )
 
-    sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
-
 
 @pytest.mark.dcos_min_version('1.11')
 @pytest.mark.region_awareness
 @sdk_utils.dcos_ee_only
-def test_region_config_update_does_not_succeed():
-    sdk_install.install(
-        config.PACKAGE_NAME,
-        config.SERVICE_NAME,
-        3,
-        additional_options={"service": {"scenario": "MULTI_REGION"}})
-
+def test_region_config_update_does_not_succeed(local_service):
     sdk_plan.wait_for_completed_deployment(config.SERVICE_NAME)
     change_region_config('Europe')
     plan = sdk_plan.get_deployment_plan(config.SERVICE_NAME)
```

Comment:
If we expect the region change to fail, would we not expect the config validation errors to show up in the error list?

Reply:
That's what `assert plan.get('errors', [])` does -- if the list is empty, the value is falsy, and the assert will fail.

Reply:
Sorry, my bad. Would it make sense to check for an explicit error message instead of relying on any error?